src/wizard/javascript/index.md

```diff
@@ -32,7 +32,7 @@ Sentry.init({
 
 The above configuration captures both error and performance data. To reduce the volume of performance data captured, change `tracesSampleRate` to a value between 0 and 1.
 
-One way to verify your setup is by intentionally sending an event that breaks your application.
+One way to verify your setup is by intentionally causing an error that breaks your application.
```

Comment:
Check this sentence may appear in other places, I have a vague remembrance of that.

Reply:
Holy cow, you weren't kidding.

Fixed the rest and also ran prettier, because why not?